services/AUTOMATIC1111/optional-deepdanbooru.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install tensorflow-cpu==2.10
+pip install tensorflow-io==0.27.0 
+pip install git+https://github.com/KichangKim/DeepDanbooru.git@edf73df4cdaeea2cf00e9ac08bd8a9026b7a7b26#egg=deepdanbooru deepdanbooru
```

Comment:
are you sure that you need to install the git repo and deepdanbooru? is the deepdanbooru at the end a mistake or it should be there?

Reply:
I don't honestly know much about pip and thought it needed something special. I was following this based on instructions over in the AUTOMATIC1111 repo.  I try to avoid python and everything to do with it like the plague.